LL/flaith_clainde_mael_ugrai.ttl

```diff
@@ -46,76 +50,89 @@
     foaf:name "Fhlaind Úallaich";
     foaf:name "Fland Úallach";
-    rel:childOf <#Diummasaig>.
+    rel:childOf <#Diummasaig>;
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#FhlaindUallaig>.
 
 <#Diummasaig>
     a foaf:Person;
     foaf:name "Diummasaig";
     foaf:name "Diummasach";
-    rel:childOf <#Fhogartaig>.
+    rel:childOf <#Fhogartaig>;
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#Diummasaig>.
 
 <#Fhogartaig>
     a foaf:Person;
     foaf:name "Fhogartaig";
     foaf:name "Fogartach";
-    rel:childOf <#Mugroin>.
+    rel:childOf <#Mugroin>;
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#Fhogartaig>.
 
 <#Mugroin>
     a foaf:Person;
     foaf:name "Mugroin";
     foaf:name "Mugron";
-    rel:childOf <#Fhlaind-dc2c8740>.
+    rel:childOf <#Fhlaind-dc2c8740>;
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#Mugroin-e4ec6d96>.
 
 <#Fhlaind-dc2c8740>
     a foaf:Person;
     foaf:name "Fhlaind";
     foaf:name "Fland";
-    rel:childOf <#Diummasaig-02276330>.
+    rel:childOf <#Diummasaig-02276330>;
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#Flaind-d033590a>.
 
 <#Diummasaig-02276330>
     a foaf:Person;
     foaf:name "Diummasaig";
-    rel:childOf <#Corcrain-0db351f0>.
+    rel:childOf <#Corcrain-0db351f0>;
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#Diummasaig-9f7ff47d>.
 
 <#Corcrain-0db351f0>
     a foaf:Person;
     foaf:name "Corcrain";
     foaf:name "Corcran";
-    rel:childOf <#MaelUgrai>.
+    rel:childOf <#MaelUgrai>;
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#Corcrain-effa7150>.
 
 <#MaelUgrai>
     a foaf:Person;
     foaf:name "Mael Ugrai";
-    rel:childOf <#Conchobuir>.
+    rel:childOf <#Conchobuir>;
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#MaelUgrai>.
 
 <#Conchobuir>
     a foaf:Person;
     foaf:name "Conchobuir";
     foaf:name "Conchobar";
-    rel:childOf <#Aeda-2f743840>.
+    rel:childOf <#Aeda-2f743840>;
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#Conchobuir-c28d4a69>.
 
 <#Aeda-2f743840>
     a foaf:Person;
     foaf:name "Aeda";
-    rel:childOf <#Tommaltaig>.
+    rel:childOf <#Tommaltaig>;
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#Aeda-e8b5d449>.
 
 <#Tommaltaig>
     a foaf:Person;
     foaf:name "Tommaltaig";
     foaf:name "Tommaltach";
-    rel:childOf <#FhlaindDaChongal>.
+    rel:childOf <#FhlaindDaChongal>;
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#Tommaltaig>.
 
 <#FhlaindDaChongal>
     a foaf:Person;
     foaf:name "Fhlaind Da Chongal";
     foaf:name "Fland Da Chongl";
-    owl:sameAs <http://example.com/LL/rig_h_falge.ttl#FhlaindDaChongal>.
+    owl:sameAs <http://example.com/LL/rig_h_falge.ttl#FhlaindDaChongal>;
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#FlaindDaChongal>.
 #owl:sameAs <http://example.com/LL/rig_h_falge.ttl#FhlaindDaChongal> ? - CY
 #Yes, I think so. The two tracts are right next to each other. EPT
```

Comment:
If you have fixed this, the comment here is probably now unnecessary so you can remove it.